packages/models/src/eservice/protobufConverterToV2.ts

```diff
@@ -119,6 +119,17 @@ export const toDescriptorV2 = (input: Descriptor): EServiceDescriptorV2 => ({
   archivedAt: dateToBigInt(input.archivedAt),
   rejectionReasons:
     input.rejectionReasons?.map(toDescriptorRejectedReasonV2) ?? [],
+  templateVersionRef: input.templateVersionRef
+    ? {
+        id: input.templateVersionRef.id,
+        interfaceMetadata: input.templateVersionRef.interfaceMetadata
+          ? {
+              ...input.templateVersionRef.interfaceMetadata,
+              serverUrls: input.serverUrls,
+            }
+          : undefined,
+      }
+    : undefined,
```

Comment:
Not needed if you remove `serverUrls` from the `TemplateInstanceInterfaceMetadataV2` protobuf

Reply:
It's a leftover , removed in [924ddbb](https://github.com/pagopa/interop-be-monorepo/pull/1643/commits/924ddbb28ea60e75df28061b73ff97d8687cfd1a)